src/pages/apiInformation/apiInformation.page.jsx

```diff
@@ -11,109 +14,151 @@ import Breadcrumbs from "../../components/breadcrumbs/breadcrumbs.component.jsx"
+                            <li><Skeleton/></li>
+                            <li><Skeleton/></li>
+                            <li><Skeleton/></li>
+                        </ul>
+                devUrl = stagingUrl = <Skeleton/>
+            }
+        }
 
         TableData.push(
             ["Version", SelectVersion],
-            ["Development Base URL", apiData.developmentBaseUrl],
-            ["Staging Base URL", apiData.stagingBaseUrl],
-            ["Relevant Links", Links.map(link => (
-                <li key={link} >   
-                    <a className="lbh-link lbh-link--no-visited-state" href={link.url} >{link.linkText} {!link.url && "(TBC)"}</a>
-                </li>
-            ))]
+            ["Development Base URL", devUrl],
+            ["Staging Base URL", stagingUrl],
+            ["Relevant Links", links]
         );
     }
-
-    const TableData = [];
-    if (isLoaded && !error){
-        formatApiData();
+    
+    if(swaggerStatus.error && apiStatus.error){
+        return(
+            <main className="lbh-main-wrapper" id="main-content" role="main">
+                <div id="api-info-page" className="lbh-container">
+                    <Error title="Oops! Something went wrong!" summary={swaggerStatus.error + " | " + apiStatus.error} /> 
+                </div>
+            </main>
+        );
     }
+    
+    const TableData = [];
+    formatApiData();
 
     return (
         <main className="lbh-main-wrapper" id="main-content" role="main">
             <div id="api-info-page" className="lbh-container">
-                {isLoaded ? ( 
-                    error ? 
-                        <Error title="Oops! Something went wrong!" summary={error.message} /> 
-                        :
-                        <>
-                            <div className="sidePanel">
-                                <Breadcrumbs />
-                                <h1>{apiData.swaggerData.info.title}</h1>
-                                <EnvironmentTags tags={apiData.swaggerData.tags && apiData.swaggerData.tags.map(tag =>(tag.name))} />
-                                <p className="lbh-body-m">{apiData.swaggerData.info.description}</p>
-                            </div>
-                            <div className="main-container table-container">
-                                    <span className="govuk-caption-xl lbh-caption">API Information</span>
-                                    <hr/>
-                                    <Table tableData={TableData} />
-                            </div>
-                        </>
-                )
-                : 
-                <h3>Loading..</h3>}
+                <div className="sidePanel">
+                    <Breadcrumbs />
+                    <h1>
+                        {!apiStatus.error && (apiData.apiName || <Skeleton/>)}
+                        {(apiStatus.error && (swaggerStatus.isLoaded ? swaggerData.info.title : <Skeleton/>))}
```

Comment:
The loading skeleton is a lot easier to use than I thought, nice library find

Reply:
@1JW1 It's really neat! And a great way to load stuff from multiple APIs with different latency, etc.